src/utils/diff/index.ts

```diff
@@ -0,0 +1,29 @@
+import * as jsdiff from 'diff';
+import debug from 'debug';
+import { isUndefined, isNull } from 'lodash';
+import { cleanUp, stringify } from './util';
+import { FAILED_TO_GENERATE_DETAILED_DIFF } from '../../notifications/information-message';
+
+const debugLogger = debug('testcafe:util:diff');
+
+function unifiedDiff (actual: string, expected: string): string {
+    const msg   = jsdiff.createPatch('string', actual, expected);
+    const lines = msg.split('\n').splice(5);
```

Comment:
I checked that first 5 string contains some redundant information, however, it would be difficult to remember this next time when I see this code. Please add a note